examples/example-pokemon-vue/po.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Let's make sure not to remove these files before merging this, like in the other example.

We could add their names to the config if we really wanted them, but I think we won't need to commit them